connector_office_365_hr_leave/models/calendar_event.py

```diff
@@ -18,4 +18,5 @@ def _office_365_from_event(self, event):
                 values['subject'] = '{} - {}'.format(
                     values['subject'], leave.name
                 )
+        values['ShowAs'] = 'away'
```

Comment:
```suggestion
        values['ShowAs'] = 'oof'
```

The value expected by office365 is 'oof'. To find it, I used the graph explorer (https://developer.microsoft.com/en-us/graph/graph-explorer), created an event on https://outlook.office.com/calendar/view/week with our zhinst account and executed a GET https://graph.microsoft.com/v1.0/me/calendar/events on graph explorer.